build/main.go

```diff
@@ -85,6 +85,14 @@ type MemoHash struct {
 	Value xdr.Hash
 }
 
+// Limit is a mutator that sets a limit on the change_trust operation
+type Limit struct {
+	Amount string
+}
+
+// MaxLimit represents the maximum value that can be passed as trutline Limit
+var MaxLimit = Limit{"922337203685.4775807"}
```

Comment:
Rather than a straight constant, what are your thoughts about using  `amount.String(math.MaxInt64)` instead?  My hunch is that it will help teach people about the relationship between the string amounts and how native amount are represented.